src/nvim/fileio.c

```diff
@@ -5452,18 +5458,14 @@ static void au_remove_pat(AutoPat *ap)
 static void au_remove_cmds(AutoPat *ap)
 {
   for (AutoCmd *ac = ap->cmds; ac != NULL; ac = ac->next) {
+    if (ac->lua_cmd != LUA_NOREF) {
+      executor_free_luaref(ac->lua_cmd);
+    }
     XFREE_CLEAR(ac->cmd);
   }
   au_need_clean = true;
 }
 
-// Delete one command from an autocmd pattern.
-static void au_del_cmd(AutoCmd *ac)
```

Comment:
This function was used only in a single place, I've figure out it's better w/o this indirection.